python/build-wheel-manylinux2014.sh

```diff
@@ -99,11 +99,11 @@ for PYTHON_NUMPY in "${PYTHON_NUMPYS[@]}" ; do
 
     # build ray wheel
     PATH="/opt/python/${PYTHON}/bin:$PATH" \
-    "/opt/python/${PYTHON}/bin/python" setup.py bdist_wheel
+    "/opt/python/${PYTHON}/bin/python" setup.py -q bdist_wheel
```

Comment:
does -q mean to be less verbose?

Reply:
> does -q mean to be less verbose?

means "quiet". it will not print all the "copying file", "adding file" lines.